h/static/styles/annotations.scss

```diff
@@ -34,7 +34,14 @@
   margin: .8em 0;
 }
 
-.annotation-header { margin-top: 0 }
+.annotation-header {
+  margin-top: 0;
+
+  &.please-sign-in {
```

Comment:
I have an elegant fix for this, but I'd like you to spend 30m to see if you can find it.